python/txt2tags.py

```diff
@@ -3327,36 +3332,36 @@ def dumpConfig(source_raw, parsed_config):
 	]
 	# First show all RAW data found
 	for label, cfg in data:
-		print _('RAW config for %s')%label
+		print(_('RAW config for %s')%label)
 		for target,key,val in cfg:
 			target = '(%s)'%target
 			key    = dotted_spaces("%-14s"%key)
 			val    = val or _('ON')
-			print '  %-8s %s: %s'%(target,key,val)
-		print
+			print('  %-8s %s: %s'%(target,key,val))
+		print()
 	# Then the parsed results of all of them
-	print _('Full PARSED config')
-	keys = parsed_config.keys() ; keys.sort()  # sorted
+	print(_('Full PARSED config'))
+	keys = list(parsed_config.keys()) ; keys.sort()  # sorted
 	for key in keys:
 		val = parsed_config[key]
 		# Filters are the last
 		if key == 'preproc' or key == 'postproc':
 			continue
 		# Flag beautifier
-		if key in FLAGS.keys() or key in ACTIONS.keys():
+		if key in FLAGS or key in ACTIONS:
 			val = onoff.get(val) or val
 		# List beautifier
 		if type(val) == type([]):
```

Comment:
While this is still valid code, I don't think it is very good coding style. Could you change all these cases as follows?

```suggestion
		if isinstance(val, list):
```